runtime/runtime-params-estimator/src/utils.rs

```diff
@@ -223,3 +225,10 @@ pub(crate) fn generate_fn_name(index: usize, len: usize) -> Vec<u8> {
     }
     name
 }
+
+/// Create a WASM module that is empty except for a main method and a single data entry with n characters
+pub(crate) fn generate_data_only_contract(data_size: usize) -> Vec<u8> {
+    let payload = rand::thread_rng().sample_iter(&Alphanumeric).take(data_size).collect::<String>();
```

Comment:
I'd rather not make this random, again, from simplicity's point of view. I'd use

```rust
"All work and no play makes Jack a dull boy".chars().cycle().take(data_size)
```

Or are we caring about this being non-comperssible? If we do, let's use a random string with a fixed seed.